loadtest/tests.go

```diff
@@ -108,6 +108,56 @@ func actionGetStatuses(c *EntityConfig) {
 	}
 }
 
+func actionLeaveJoinChannel(c *EntityConfig) {
+	team, channel := c.UserData.PickTeamChannel(c.r)
+
+	if team == nil || channel == nil {
+		return
+	}
+
+	channelId, err := c.GetTeamChannelId(team.Name, channel.Name)
+	if err != nil {
+		mlog.Error("Unable to get channel from map", mlog.String("team", team.Name), mlog.String("channel", channel.Name), mlog.Err(err))
+		return
+	}
+
+	userId := ""
+	if user, resp := c.Client.GetMe(""); resp.Error != nil {
+		mlog.Error("Failed to get me", mlog.Err(resp.Error))
+		return
+	} else {
+		userId = user.Id
+	}
+
+	if _, resp := c.Client.GetChannel(channelId, ""); resp.Error != nil {
+		mlog.Error("Failed to get channel", mlog.String("channel_id", channelId), mlog.Err(resp.Error))
+		return
+	}
+
+	removed, _ := c.Client.RemoveUserFromChannel(channelId, userId)
+
```

Comment:
Not a huge fan of the code repetition in here. Maybe better to just check if the user is already in the channel and proceed accordingly? 2/5 on this

Reply:
I agree about the code repetition and initially I wanted to write the check you suggest. However I feared that adding it would somehow "alter" the test especially since it's not something that would normally run when a user joins or leaves a channel from the webapp.

Reply:
@streamer45 - I think we're doing quite alot of other "non-normal" stuff in the load tests, so that doesn't worry me as much :)
@lieut-data thoughts?

Reply:
Ideally, the loadtest actions map closely to the corresponding actions in the webapp. I share @streamer45's concern that an extra API call meant only to juggle internal state in the loadtest will skew the results.

I do think the "reversal" is a clever approach. The duplication doesn't bother me too much, though I think we can reduce at least one call ala:
```go
// try remove
// always add
// if !removed earlier, remove
```